caddyhttp/httpserver/mitm.go

```diff
@@ -360,6 +363,58 @@ func (info rawHelloInfo) looksLikeFirefox() bool {
 	return assertPresenceAndOrdering(expectedCipherSuiteOrder, info.cipherSuites, false)
 }
 
+// looksLikeChromeOniOS returns true if info looks like a handshake
+// from a modern version of Chrome on iOS.
+func (info rawHelloInfo) looksLikeChromeOniOS() bool {
```

Comment:
What if we just called looksLikeChrome and then if false, checked for the one difference between the two which is the existence of that one cipher suite? Rather than duplicating almost the whole function.